source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import domify from 'doma';
+import elementReady from 'element-ready';
+
+import features from '.';
+
+async function init(): Promise<void | false> {
+	const currentFeature = /features\/(.*).[a-z]{3}$/.exec(location.pathname)![1];
```

Comment:
Prefer negating characters when possible instead if a generic `.`. Also probably you want `+` because `*` matches also 0 characters
```suggestion
	const [, currentFeature] = /features\/([^.]+)/.exec(location.pathname)!;
```